RetailCoder.VBE/UI/RubberduckUI.de.resx

```diff
@@ -908,13 +908,13 @@ Warnung: Alle eigenen Einstellungen gehen dabei verloren. Die Originaldatei wird
     <value>Standard Repositorypfad</value>
   </data>
   <data name="SourceControl_EmailAddressLabel" xml:space="preserve">
-    <value>Email Adresse</value>
+    <value>E-Mail-Adresse</value>
   </data>
   <data name="SourceControl_GlobalSettings" xml:space="preserve">
     <value>Globale Einstellungen</value>
   </data>
   <data name="SourceControl_IgnoreFile" xml:space="preserve">
-    <value>Datei ignorieren</value>
+    <value>'.gitignore' Datei</value>
```

Comment:
disregard that then :+1: Thanks for the well-done fix @INOPIAE 

that's the problem of translating only the text you get :neutral_face: